ui/color/BUILD.gn

```diff
@@ -3,17 +3,39 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at https://mozilla.org/MPL/2.0/.
 
-copy("copy_leo_color_headers") {
-  sources = [ "//brave/node_modules/@brave/leo/tokens/skia/colors.h" ]
-  outputs = [ "$root_out_dir/gen/brave/ui/color/leo/colors.h" ]
+copy("copy_nala_color_ids") {
+  sources = [ "//brave/node_modules/@brave/leo/tokens/skia/nala_color_id.h" ]
+
+  outputs = [ "$root_out_dir/gen/brave/ui/color/nala/{{source_file_part}}" ]
+}
+
+source_set("nala_color_id") {
+  sources = get_target_outputs(":copy_nala_color_ids")
+
+  deps = [ ":copy_nala_color_ids" ]
+}
+
+copy("copy_nala_color_mixer") {
+  sources = [
+    "//brave/node_modules/@brave/leo/tokens/skia/nala_color_mixer.cc",
+    "//brave/node_modules/@brave/leo/tokens/skia/nala_color_mixer.h",
+  ]
+
+  outputs = [ "$root_out_dir/gen/brave/ui/color/nala/{{source_file_part}}" ]
```

Comment:
I don't think this is correct, because in cross-compile mode `gen` may be located one level deeper. Shouldn't this be `$target_gen_dir/{{source_file_part}}` instead? You may need to rename the target to `nala` for things to work.

same for the above target (not sure why these are separate targets, just merge them into one, copying is not cpu-intensive operation during the build, so it can just be a single target).

Reply:
Wow, didn't know about the `gen` folder being located in different places, but it makes sense! I've replaced it with `$root_gen_dir` instead, which I think should work :smile:

Reply:
> Wow, didn't know about the `gen` folder being located in different places, but it makes sense! I've replaced it with `$root_gen_dir` instead, which I think should work, and merged the copy targets 😄

have you tried `$target_gen_dir`? GN should fill it with `gen/brave/ui/color/{target_name}` path for you.